src/v/crypto/README.md

```diff
@@ -154,3 +154,8 @@ buffer with random data.  Both functions also have a flag to indicate if the cal
 wants to use the 'private' DRBG.  For more information on this please refer to
 OpenSSL's [documentation](https://www.openssl.org/docs/man3.0/man3/RAND_priv_bytes.html)
 on this.
+
+## Base64 Decoding
+
+To decode base64 data, simply use `crypto::base64_decode()` to get back a
+`std::vector<uint8_t>` containing the decoded data.
```

Comment:
Why is base64 part of the crypto library? Also, we have an optimized implementation of base64 in the tree.

Reply:
> Why is base64 part of the crypto library

I saw that [jwt.h](https://github.com/redpanda-data/redpanda/blob/bb6a83ef8659a11b42fd21070b2f4937e645b1e1/src/v/security/jwt.h#L104-L118) was using CryptoPP's base64 implementation so I added it to the library.

> we have an optimized implementation of base64 in the tree.

Did not know that

Reply:
utils/base64.h

Reply:
Ah further clarification - the JWT stuff needs to decode URL safe Base64 strings, that is strings that use `-` and `_` rather than `+` and `/`

Reply:
should we instead add an interface for dealing with url encoded base64 strings to the utils/base64.h interface?

Reply:
yeah that's probably what we'll do... I'll drop the base64 commit